bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/VoiceManager.java

```diff
@@ -492,6 +513,26 @@ public STTService getSTT() {
     }
 
     /**
+     * Retrieves a STT service with the given id.
+     *
+     * @param id the id of the STT service
+     *
+     * @return a STT service or null, if no service with this id exists
+     */
+    public STTService getSTT(String id) {
+        return sttServices.get(id);
+    }
+
+    /**
+     * Retrieves all STT services.
+     *
+     * @return a set of STT services
```

Comment:
* it is not necessary a "set" but a collection
* should we add the information, that the returned collection will throw an exception if someone tried to modify it (so add it is an unmodifiable collection)